static/js/selectors/websites.ts

```diff
@@ -12,6 +12,24 @@ export const getWebsiteDetailCursor = createSelector(
   websiteDetails => memoize((name: string) => websiteDetails[name])
 )
 
+export const getWebsiteListingCursor = createSelector(
+  (state: ReduxState) => state ?? {},
+  state =>
+    memoize((offset: number) => {
+      const listing = state.entities?.websitesListing ?? {}
+      const response = listing[offset] ?? {}
+      const names = response?.results ?? []
+      const sites = names.map((name: string) =>
+        getWebsiteDetailCursor(state)(name)
+      )
+
+      return {
+        ...response,
+        results: sites
+      }
+    })
+)
+
```

Comment:
```suggestion
export const getWebsiteListingCursor = createSelector(
  (state: ReduxState) => state.entities?.websitesListing ?? {},
  getWebsiteDetailCursor,
  listing, websiteDetailCursor => {
    memoize((offset: number) => {
      const response = listing[offset] ?? {}
      const names = response?.results ?? []
      const sites = names.map(websiteDetailCursor)

      return {
        ...response,
        results: sites
      }
    })
  )
)
```

this is how I'd suggest doing it - reselect lets us compose selectors in a pretty fluid way like this

Reply:
Oh neat